repository/kafka/docs/v0.1.1/install.md

```diff
@@ -0,0 +1,42 @@
+# Installing KUDO Kafka operator
+
+Requirements:
+
+- [Install KUDO controller](https://kudo.dev/docs/getting-started/)
+- [Install KUDO cli](https://kudo.dev/docs/cli/)
+
+
+## Installing the Operator
+
+#### Install the Zookeeper 
+```
+kubectl kudo install zookeeper --instance=zk
+```
+
+#### Install the Kafka 
+```
+kubectl kudo install kafka --instance=kafka
```

Comment:
so by default the connection strings etc. from kafka to ZK work when no parameters are overriden on both kafka and zk?

Reply:
it tries with default parameters if zk is also installed in `default` ns.
We can be more specific in the configuration section maybe